duckduckgo/iOS
# Upgrade autoconsent to 5.3.0

Task/Issue URL: https://app.asana.com/0/1177771139624306/1205239910258201/f
Tech Design URL:
CC: @tomasstrba @kdzwinel 

**Description**:
There was an issue in autoconsent affecting older webkit versions. autoconsent 5.3.0 contains a fix for it. See also a [macos PR](https://github.com/duckduckgo/macos-browser/pull/1460)



**Steps to test this PR**:
1. run the app on the oldest supported macOS/WebView version
2. enable CPM
3. confirm that CPM works (e.g. on https://doodle.com/)
4. confirm no CPM exceptions on that and other pages



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
